src/views/HomeView.vue

```diff
@@ -3,31 +3,32 @@
     <h1 class="search-title">
       Wie entsorge ich im Landkreis Konstanz &hellip;
     </h1>
-    <searchbar :items="materials" class="my-5" />
+    <material-searchbar :items="materials" class="my-5" />
     <v-row class="metrics">
       <v-col><material-metric-card /></v-col>
       <v-col><location-metric-card /></v-col>
       <v-col><tipp-metric-card /></v-col>
     </v-row>
     <tip-card v-if="tip && tip.id" :tip="tip" />
     <v-skeleton-loader v-else type="card" />
+    <h2>Unsere Partner</h2>
     <partner-carousel />
   </v-container>
 </template>
 
 <script>
-import Searchbar from '@/components/searchbar/Searchbar'
+import MaterialSearchbar from '@/components/searchbar/MaterialSearchbar'
 import PartnerCarousel from '@/components/partner/PartnerCarousel'
 import TipCard from '@/components/tips/TipCard'
 import MaterialMetricCard from '@/components/metric/MaterialMetricCard.vue'
 import LocationMetricCard from '@/components/metric/LocationMetricCard.vue'
 import TippMetricCard from '@/components/metric/TippMetricCard.vue'
 
 export default {
-  name: 'Home',
+  name: 'HomeView',
```

Comment:
Ich glaube den `name` in Views kann man hier weglassen, da der sich ja glaub den Namen aus dem `router` nimmt